src/shared/modules/dbMeta/dbMetaEpics.ts

```diff
@@ -198,8 +202,14 @@ async function getNodeAndRelationshipCounts(store: any) {
           relationships
         })
       )
+      const summary = res.summary as ResultSummary
+      return {
+        requestSucceeded: true,
+        timeTaken: summary.resultAvailableAfter.toNumber()
```

Comment:
Before merging we need to test against a big dataset with the RBAC issue so we're sure it still works.

Reply:
Tested //noah